harvester/edurep/tests/test_get_harvest_seeds.py

```diff
@@ -3,11 +3,12 @@
 from django.test import TestCase
```

Comment:
In deze file test je alleen de surfsharekit bron, klopt dat? De andere bronnen worden vermoedelijk ergens anders getest neem ik aan.

Reply:
Nee, we testen eigenlijk alleen Sharekit materialen uit Edurep. Er is niet een cross-sectie van bronnen voor Edurep